BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1003,7 +1005,12 @@ public void MissileLaunch()
 
                 //TARGETING
                 startDirection = transform.forward;
-
+                launchPosition = transform.position;
```

Comment:
`launchPosition` will need updating in `FloatingOriginCorrection`.

Reply:
Making `FloatingOriginCorrection` in MissileBase.cs `virtual` and adding
```csharp
public override void FloatingOriginCorrection()
{
    base.FloatingOriginCorrection();
    if (HasFired && !HasExploded && BDKrakensbane.IsActive)
        launchPosition -= BDKrakensbane.FloatingOriginOffsetNonKrakensbane;
}
```
to MissileLauncher.cs should do this. (I've added this locally in the pr507 branch.)